projects/optic/src/commands/oas/capture.ts

```diff
@@ -307,6 +333,7 @@ async function renderCaptureProgress(
     }
   }
 
+  clearTimeout(timer);
```

Comment:
oh yeah good call.

Reply:
well actually it also needs to clear after the stream ends. Within the timeout I check the number of interactions. If you exit before 13 seconds with no traffic captured, putting it on L330 will hang the process 13 - seconds since started

Reply:
I see, this works, you could also put it in L330 and then remove the `interactionCount` check